specification/devcenter/DevCenter/DevBox/routes.tsp

```diff
@@ -279,6 +279,223 @@ interface DevBoxes {
+      """)
+      @maxLength(36)
+      @minLength(2)
+      @pattern("^[a-zA-Z0-9]{8}-([a-zA-Z0-9]{4}-){3}[a-zA-Z0-9]{12}$|^me$")
+      @path
+      userId: string;
+
+      @doc("The name of a Dev Box.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      devBoxName: string;
+
+      @doc("Optional query parameter to specify what properties should be included in the response.")
+      @query
+      include?: ListCustomizationGroupsIncludeProperty;
+    },
+    CustomizationGroupListResult
+  >;
+
+  @doc("""
+    Gets a customization group.
+  """)
+  @added(APIVersions.v2024_05_01_preview)
+  getCustomizationGroup is StandardResourceOperations.ResourceRead<CustomizationGroup>;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Standard operations produce wrong swagger"
+  // The standard operation names the body as resource, so the generated swagger has parameter
+  //  {
+  //     "name": "resource",
+  //     "in": "body",
+  //  },
+  // instead of the desired
+  //  {
+  //     "name": "body",
+  //     "in": "body",
+  //  }
+  @doc("Applies customizations to the Dev Box.")
+  @put
+  @route("/projects/{projectName}/users/{userId}/devboxes/{devBoxName}/customizationGroups/{customizationGroupName}")
+  @added(APIVersions.v2024_05_01_preview)
+  createCustomizationGroup is Azure.Core.Foundations.Operation<
+    {
+      @doc("The DevCenter Project upon which to execute operations.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      projectName: string;
+
+      @doc("""
+        The AAD object id of the user. If value is 'me', the identity is taken from the 
+        authentication context.
+      """)
+      @maxLength(36)
+      @minLength(2)
+      @pattern("^[a-zA-Z0-9]{8}-([a-zA-Z0-9]{4}-){3}[a-zA-Z0-9]{12}$|^me$")
+      @path
+      userId: string;
+
+      @doc("The name of a Dev Box.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      devBoxName: string;
+
+      @doc("A customization group name.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      customizationGroupName: string;
+
+      @doc("Represents the body request of a Dev Box creation. Dev Box Pool name is required. Optionally set the owner of the Dev Box as local administrator")
+      @body
+      body: CustomizationGroup;
+    },
+    CustomizationGroup
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "No model for this."
+  @doc("Gets the log for a customization task.")
+  @route("/projects/{projectName}/users/{userId}/devboxes/{devBoxName}/customizationGroups/{customizationGroupName}/logs/{customizationTaskId}")
+  @get
+  @added(APIVersions.v2024_05_01_preview)
+  getCustomizationTaskLog is Azure.Core.Foundations.Operation<
```

Comment:
Could this operation be a resource read?

Reply:
As far as I'm aware no, there's no model to pass in to StandardResourceOperations.ResourceRead< **here** >;